modules/_labs/core/react/stories/stories_visualTesting.tsx

```diff
@@ -0,0 +1,115 @@
+/// <reference path="../../../../../typings.d.ts" />
+/** @jsx jsx */
+import {jsx} from '@emotion/core';
+import styled from '@emotion/styled';
+import {spacing} from '@workday/canvas-kit-react-core';
+import type, {space} from '..';
+import {withSnapshotsEnabled} from '../../../../../utils/storybook';
+
+export default withSnapshotsEnabled({
+  title: 'Testing|React/Labs/Core',
+});
+
+export const Header = () => (
+  <section>
+    <h1 style={type.brand1}>Brand 1 Header</h1>
+    <h2 style={type.brand2}>Brand 2 header</h2>
+    <h1 style={type.h1}>H1 Header</h1>
+    <h2 style={type.h2}>H2 Header</h2>
+    <h3 style={type.h3}>H3 Header</h3>
+    <h4 style={type.h4}>H4 Header</h4>
+    <h5 style={type.h5}>H5 Header</h5>
+  </section>
+);
```

Comment:
Side note: Working on these stories has made me much more in favor of creating `<Heading />` and `<Text />` as primitive components.

Reply:
We have `H1`, `H2`, etc. just not `Text`. Have a look at the latest changes to the non-labs core stories